pkg/kernel/patches-5.10.x/0050-LF-3689-2-net-stmmac-let-PHY-keep-RXC-clock-on-durin.patch

```diff
@@ -11,25 +11,23 @@ AR8031 PHY doesn't implement this feature, RTL8211FD PHY does.
 This patch fixes below error log during suspend/resume test with WoL enabled.
 imx-dwmac 30bf0000.ethernet eth0: Timeout accessing MAC_VLAN_Tag_Filter
 
-Reviewed-by: Jun Li <jun.li@nxp.com>
```

Comment:
You should keep these lines (reviewed-by, signed-off, etc), so we don't lost traceability.

Reply:
But my changes haven't been reviewed nor signed by those people ...

Reply:
You can describe what you did just below the last Reviewed-by and put your own Signed-off-by, for example 

```
imx-dwmac 30bf0000.ethernet eth0: Timeout accessing MAC_VLAN_Tag_Filter

Reviewed-by: Jun Li <jun.li@nxp.com>

Ported to the kernel version A.B.C.

Signed-off-by: Max Mustermann

```